refactor: revert default content to live in host MFE (#41)
openedx/frontend-plugin-framework: README.rst

.. _JavaScript-based environment configuration: https://github.com/openedx/frontend-platform/blob/master/docs/decisions/0007-javascript-file-configuration.rst
.. _Promote JavaScript file configuration and deprecate environment variable configuration: https://github.com/openedx/frontend-platform/blob/master/docs/decisions/0007-javascript-file-configuration.rst

Priority
````````

The priority property determines where the widgets should be placed based on a 1-100 scale. A widget with a priority of 10
will appear above a widget with a priority of 20.

Default Content
```````````````

> The component that is wrapped by a Plugin Slot is referred to as the "default content". In order to render the content,

Reviewer: Nit:

```suggestion
The component that is wrapped by a Plugin Slot is referred to as the "default content". In order to render this content,
```

the ``keepDefault`` boolean in the slot should be set to ``true``. For organizations who aren't using the Plugin Slot

> (and thus don't define the slot via JS config), keepDefault will default to ``true``, thus ensuring that the developer

Reviewer: Nit:

```suggestion
(and therefore aren't defining a slot via JS config), ``keepDefault`` will default to ``true``, thus ensuring that the developer
```

> experience doesn't change other than having a Plugin Slot wrapper around the component.

Reviewer: This is a nit pick to go along with the previous one. I found this to flow a little better but I'm curious what your thoughts are

```suggestion
experience isn't affected; the only change to note being the component wrapped in a PluginSlot.
```

Author: I agree that this sentence could be a little clearer, so I rewrote it with some of your suggestion!

> To use the Wrap or Modify operation on default content, the ``widgetId`` you would use to refer to the content is ``defaults_contents``.

Reviewer: Make this adjustment because we may add more operations and this also didn't account for the "hide" operation

```suggestion
If you need to use a plugin operation (e.g. Wrap, Hide, Modify) on default content, the ``widgetId`` you would use to refer to the content is ``defaults_contents``.
```
